### DISCUSSION TITLE
Fooocus will only work with Default model

### DISCUSSION THREAD

**Discussion by KazraelX** [lllyasviel/Fooocus#3693] [Q&A] [answered]
I have been using Fooocus as my primary AI generator for many months now. For some reason it will now only work with the Default model. I don't think it's a bug with Fooocus because it was working fine one day and not the next. I couldn't find any changes that happened to Windows but who knows with that OS. 

Reinstalled Fooocus. Same thing. Most often it will just close the browser window and a few times has given the Connection Error in the Fooocus UI. 

Currently have tried different browsers. Disabled MS Antidefender things. Went through the troubleshooting here and still have the same problem. Made sure the swap was large enough, all that. 

I am running an older machine,  graphics card is just a 1080, 8 Gb. The head scratcher is it was working one day and not the next. 

Before I delve into posting system logs here and all that I am just wondering if anyone else has had Foocus work perfectly fine with the default model (run.bat) but not any other models.

**Comment by mashb1t** [comment, marked as answer]
Please check out the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md).
